include/range/v3/at.hpp

```diff
@@ -40,7 +40,7 @@ namespace ranges
             auto operator()(Rng &&rng, range_difference_type_t<Rng> n) const ->
                 decltype(begin(rng)[n])
             {
-                if(n >= ranges::distance(rng))
+                if( (n < 0) or (n >= ranges::distance(rng)) )
```

Comment:
range-v3 prefers to use the primary tokens (`||`) over their alternatives (`or`). We also don't typically parenthesize subexpressions unnecessarily or use spaces-inside-parens.